resources/stylesheets/base.css

```diff
@@ -1,4 +1,5 @@
 /* Base styling sheet for standard tags and common classes */
```

Comment:
regarding the filename: isn't `css/` a common convention? Spelling out "stylesheets" is less common afaik.

Reply:
Yeah it is convention and I guess stylesheets is not that much more informative than css.